student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize()
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] The constructor should have all the parameters of the parent class and with `classroom` as an extra parameter. Please add all parameters in the initialize method and add parameters of the super class in the super method as well.